lib/clients/graphql/storefront_client.ts

```diff
@@ -41,13 +41,24 @@ export class StorefrontClient extends GraphqlClient {
 
   protected getApiHeaders(): HeaderParams {
     const sdkVariant = LIBRARY_NAME.toLowerCase().split(' ').join('-');
+    const privateToken =
+      this.storefrontClass().config.privateAppStorefrontAccessToken;
+
+    if (this.storefrontAccessToken && privateToken) {
+      logger(this.storefrontClass().config).warning(
+        'You have both private and public storefront access tokens. The private token will be used.',
```

Comment:
Logging here instead of earlier at `validateConfig` because the config params doesn't include the public access token at that point. I could do a blanket log.info at the earlier stage but I feel like that's a little bit spammy

Reply:
I agree - this shouldn't be a super common case but it's nice to nudge folks to fix it!